backend/coins/btc/handlers/handlers.go

```diff
@@ -493,7 +493,11 @@ func (handlers *Handlers) postAccountSendTx(r *http.Request) (interface{}, error
 		return map[string]interface{}{"success": false, "aborted": true}, nil
 	}
 	if err != nil {
-		handlers.log.WithError(err).Error("Failed to send transaction")
+		if strings.Contains(err.Error(), "min relay fee not met") {
```

Comment:
I feel like this string could go in a constant somewhere, I'm just not sure where. Open to suggestions!

Reply:
Looking at error of the source, it seems min relay fee is not the only error where the raw tx will appear, it could be any error. 

It seems more robust to actually filter out the raw tx than match one specific error, like you suggested.

In this case you could move this logic to https://github.com/BitBoxSwiss/bitbox-wallet-app/blob/master/backend/coins/btc/electrum/client.go#L117, closer to the source where the rawTxHex is already available.